README.md

```diff
@@ -23,10 +23,21 @@ specification to describe this ability as well, and move the document to the sta
   * todo
 
 ## Goals
-  * todo
+* Enable a web app to register tasks to be run periodically, with a retry logic baked in, which has an exponential backoff.
+* Provide the web app a way to query the list of previously registered tasks.
+* Enable the web app to unregister previously registered tasks.
```

Comment:
To the first three points: rather than justifying the individual APIs (particularly `getIds()` and `unregister()` are more for developer convenience), focus on the high-level goals of the functionality itself.

Why would we want a website to be able to periodically update itself? Some ideas:

* Native apps have the ability to offer fresh content to users, even when they're offline (e.g. during a commute), improving the user experience. Why shouldn't the Web be able to do this?
* Websites can already use push notifications for updating content, but the timing of that is decided by the developer. Periodic sync allows the browser to have a say about timing - what benefit does that bring?

Reply:
We've already covered this in use cases then. Do you see any benefit in having this section if we're going to repeat the use cases?

Reply:
Let me know if you have any more thoughts here.